src/pages/posts/css/du-css-evolutif/index.md

```diff
@@ -0,0 +1,85 @@
+Modulaire, maintenable, réutilisable, accessible. Ces termes sont au cœur de nos métiers et on nous les rabache toute la journée.
+
+Malgré la grossièreté de ces mots pour certains d'entre nous, il est tout a fait possible de pondre du CSS en utilisant tous ces principes.
+
+## Maintenable
+
+L'un des trucs les plus chiants du CSS c'est bien la cascade. Pour pallier ce problème, BEM est arrivé. Il en existe plusieurs variantes et d'ailleurs on a déjà écrit un [article](http://putaindecode.fr/posts/css/petite-definition-bem/) pour en expliquer la définition.
```

Comment:
Voilà ma proposition pour ce paragraphe "L'un des trucs les plus chiants du CSS c'est bien la cascade, qui, malgré sa bonne volonté, n'apporte pas que des solutions. Pour pallier aux problèmes que l'on peut vite créer, [la méthodologie BEM](http://putaindecode.fr/posts/css/petite-definition-bem/) a vu le jour avec son [lot de variantes](http://blog.kaelig.fr/post/48196348743/fifty-shades-of-bem)."